crates/forge/src/test_execution_syscall_handler.rs

```diff
@@ -651,45 +649,26 @@ fn execute_syscall(
         DeprecatedSyscallSelector::ReplaceClass => Err(HintError::CustomHint(Box::from(
             "Replace class can't be used in tests".to_string(),
         ))),
-        DeprecatedSyscallSelector::GetBlockHash => {
-            execute_get_block_hash(
-                vm,
-                &mut system_ptr.clone(),
-                cheatable_syscall_handler.cheatnet_state,
-            )?;
-            Ok(())
-        }
         _ => cheatable_syscall_handler.execute_hint(vm, exec_scopes, hint_data, constants),
     }
 }
 
-fn execute_get_block_hash(
-    vm: &mut VirtualMachine,
-    system_ptr: &mut Relocatable,
-    _cheatnet_state: &CheatnetState,
-) -> Result<(), HintError> {
-    let _selector = stark_felt_from_ptr(vm, system_ptr)?;
-    let SyscallRequestWrapper {
-        gas_counter,
-        request: _,
-    } = SyscallRequestWrapper::<GetBlockHashRequest>::read(vm, system_ptr)?;
-
-    let sc_response = SyscallResponseWrapper::Success {
-        gas_counter: gas_counter - GET_BLOCK_HASH_GAS_COST,
-        response: GetBlockHashResponse {
-            block_hash: BlockHash(StarkFelt::from(0_u32)),
-        },
-    };
-    sc_response.write(vm, system_ptr)?;
-    Ok(())
+struct CallContractArgs {
+    _selector: Felt252,
+    gas_counter: usize,
```

Comment:
Why is it `usize` and not other type?

Reply:
I've encountered gas counters with u64/f64 values so far 😬

Reply:
Good call, i just copied existing code here but indeed we should parse u64 https://github.com/orgs/foundry-rs/projects/3/views/2?pane=issue&itemId=40983148